css/style.css

```diff
@@ -0,0 +1,82 @@
+@import url("https://fonts.googleapis.com/css2?family=Poppins:wght@400;700&display=swap");
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  border: 0;
+}
+
+body {
+  background-color: #e5e5e5;
+  width: 100%;
+  height: 100%;
+  position: absolute;
+}
+
+header {
+  display: flex;
+  justify-content: space-between;
+  padding: 54px 20px;
+  background-color: #fff;
+  background-position: center;
+  background-repeat: no-repeat;
+}
+
+.my-logo {
+  color: #6070ff;
+  font-family: "Poppins", sans-serif;
+  font-weight: 700;
+  font-size: 19px;
+  line-height: 20px;
+  text-decoration: none;
+}
+
+.headline {
+  background-color: #fff;
+  background-image: url(../images/mobile-version-bg.png);
```

Comment:
- [ ] Good job fixing the broken background Image, kindly make sure it's responsive in order to work properly as requested by the previous reviewer :recycle: 

![resp](https://user-images.githubusercontent.com/97834160/212031596-fb4a804f-d046-4b88-b32f-e51b95b9309e.png)